README.rst

```diff
@@ -3,56 +3,78 @@ openedx-completion-aggregator
 
 |pypi-badge| |travis-badge| |codecov-badge| |pyversions-badge| |license-badge|
 
-openedx-completion-aggregator is a Django app that aggregates block level
-completion data for different block types for Open edX.
+openedx-completion-aggregator is a Django app that aggregates block level completion data for different block types for Open edX.
 
-Overview
---------
+What does that mean?
 
-openedx-completion-aggregator uses the pluggable django app pattern to
-ease installation.  To use in edx-platform, do the following:
+A standard Open edX installation can track the completion of individual XBlocks in a course, which is done using the `completion library <https://github.com/edx/completion#completion>`_. This completion tracking is what powers the green checkmarks shown in the course outline and course navigation as the learner completes each unit in the course:
```

Comment:
We might want to change [this line](https://github.com/open-craft/openedx-completion-aggregator/blob/master/tox.ini#L5) to something like `ignore=D001` to make `doc8` happy in the CI.